openlibrary/plugins/worksearch/autocomplete.py

```diff
@@ -96,6 +96,7 @@ class languages_autocomplete(delegate.page):
     def GET(self):
         i = web.input(q="", limit=5)
         i.limit = safeint(i.limit, 5)
+        web.header("Cache-Control", "max-age=%d" % (24 * 3600))
```

Comment:
There is little value in caching this ; it's very unlikely a user will type the same thing twice! In general, it's not super likely a user will type the same thing many times. I imagine a veeeeeery tiny percentage of user searches will be cache hits; most will be misses. I think we might want to roll this back, and cache it only in the case we care about.

Reply:
@cdrini this comes from a librarian commenting of the lag that happens when searching languages. I don't know what exactly they're editing that they see it often enough to comment.

How about after the deploy goes how to we check how many requests this endpoint is getting on average? Also we can follow up with that librarian to see what their use case it.

Reply:
Btw this caching works across pages so if someone edits one book to add a language the next time they do that on any book it'll use the cache.